tests/integration/test_symlink.py

```diff
@@ -124,3 +133,158 @@ def test_symlinkToFileDir(self):
         res = self.watchmanCommand('query', root, expr)
         self.assertTrue(res['files'] and ('symlink_target' not in res['files'][0] or res['files'][0]['symlink_target'] is None))
         self.assertEqualUTF8Strings('d', res['files'][0]['type'])
+
+    @unittest.skipIf(os.name == 'nt', 'win')
+    def test_watchSymlinkTarget(self):
+        def make_roots():
+            rootA = self.makeRootAndConfig()
+            rootB = self.makeRootAndConfig()
+            rootC = self.makeRootAndConfig()
+            self.touchRelative(rootA, 'file')
+            self.touchRelative(rootB, 'file')
+            self.touchRelative(rootC, 'file')
+            os.symlink(os.path.join(rootC, 'file'), os.path.join(rootB, 'link'))
```

Comment:
Please add a comment something like `B/link -> C/file` so that it is quick and easy to understand the link structure at a glance :-)